src/harness/reference_models/move_list_calc/test_data/RegistrationRequest_1.json

```diff
@@ -0,0 +1,25 @@
+{
+  "fccId": "321cba_1",
+  "cbsdCategory": "A",
+  "callSign": "KPPP",
+  "userId": "John Doe",
+  "airInterface": {
+    "radioTechnology": "E_UTRA",
+    "supportedSpec": "LTE-Rel10"
+  },
+  "cbsdInfo": {
+    "vendor": "Equipment Maker C"
+  },
+  "cbsdSerialNumber": "4321dcba_1",
+  "measCapability": [ "EUTRA_CARRIER_RSSI_ALWAYS" ],

```

Comment:
FYI this meas. capability value is no longer supported. To make things easier, you could simply use [ ], which is what we use in many of the other test cases.